Drone/drone_controller.py

```diff
@@ -28,18 +29,20 @@ def __init__(self, drone):
 
         self._logger = logging.getLogger(__name__)
         self._debug = True
-
         self.drone = drone
+        self._instruction_queue = PriorityQueue()
 
         coloredlogs.install(level='DEBUG')
```

Comment:
Consider making an enum for logging levels or using the built-in logging mode constants if they achieve the same effect here.